browser_history/cli.py

```diff
@@ -3,11 +3,13 @@
 
 import sys
 import argparse
-from browser_history import get_history, generic, browsers
+import os
+from browser_history import get_history, get_bookmarks, generic, browsers
 
 # get list of all implemented browser by finding subclasses of generic.Browser
 AVAILABLE_BROWSERS = ', '.join(b.__name__ for b in generic.Browser.__subclasses__())
 AVAILABLE_FORMATS = ', '.join(generic.Outputs.formats)
+AVAILABLE_TYPES='history,bookmarks,all'
```

Comment:
Just a suggestion - add spaces here to make it look better in the help page

Reply:
yuup